src/instruments/src/util.js

```diff
@@ -78,14 +78,6 @@ export function setSimVar(name, value, type = SIMVAR_TYPES[name]) {
     return SimVar.SetSimVarValue(name, type, value);
 }
 
-export function useSimVar(name, type) {
-    const [value, updater] = useState(0);
-
-    useUpdate(() => updater(getSimVar(name, type)));
-
-    return value;
-}
-
 export function useGlobalVar(name, type) {
```

Comment:
**question**: `react-msfs` contains `useGlobalVar` now. Can this be removed?

Reply:
> 
> 
> Thanks for your work.
> 
> In reply to @tracernz: in some ways `react-msfs` was extracted too soon. It would have been better to keep it as a separate project within the same repo for now and use TS paths to link `@flybywiresim/react-msfs` to that repo. (And don't forget to add a module resolver configuration, see example [here](https://github.com/flybywiresim/a32nx/pull/5359/files#diff-8054277b56da998d1c093401bfdc6dff087be6b1cb22c10e93903a185aa62304)).
> 
> I'd like to note that the README on `react-msfs` itself doesn't make any mention of the hooks. So having them in there might be questionable.
> 
> Lastly I'd like to mention that many of the SimVar hook stuff was created by theomessin in collaboration with beheh under GPLv3 license. Although the repo later did briefly change to MIT, I do hope we have asked both for permission to extract it into a library under the MIT license.

`react-msfs` was supposed to be changed to GPLv3. Forgot that (I already have permission from the other authors of the library). Will mark as draft until that is done.

Reply:
As for the hooks in the README, I was going to draft up one but noticed that the bit of doc there was actually supposed to be in the map component folder. I will move the existing README and write a new one mentioning the hooks.

Reply:
> **question:** react-msfs contains useGlobalVar now. Can this be removed?

Yes. Will fix.